src/app/zap-templates/templates/app/CHIPClusters-src.zapt

```diff
@@ -18,11 +18,13 @@ namespace Controller {
 // TODO(#4503): length should be passed to commands when byte string is in argument list.
 // TODO(#4503): Commands should take group id as an argument.
 
-{{#chip_client_clusters}}
-
-// {{asUpperCamelCase name}} Cluster Commands
-{{#chip_cluster_commands}}
-CHIP_ERROR {{asUpperCamelCase clusterName}}Cluster::{{asUpperCamelCase name}}(Callback::Cancelable * onSuccessCallback, Callback::Cancelable * onFailureCallback{{#chip_cluster_command_arguments_with_structs_expanded}}{{#if_is_struct type}}{{else}}, {{chipType}} {{asLowerCamelCase label}}{{/if_is_struct}}{{/chip_cluster_command_arguments_with_structs_expanded}})
+{{#all_user_clusters_with_outgoing_commands}}
```

Comment:
Now that `CHIPClusters.zapt` uses back `chip_client_clusters` can we use it here too for consistency and couple it with `{{#all_outgoing_commands_for_cluster}}` ?

Reply:
I can change it if you want but we get those 1 line comments with no corresponding code when there are no outgoing commands from client which seems confusing. If we still want it then I can make a change for that.

Reply:
If we put the comments inside a `{{#first}}` block we would only get them when we have something to show, no?

Reply:
Makes sense. yes that is correct. Will switch the cluster loop back

Reply:
Using chip_client_clusters now